Auxiliary files are provided by the developer/build/cmake package.

--- cmake-3.31.10/CMakeLists.txt.orig
+++ cmake-3.31.10/CMakeLists.txt
@@ -553,9 +553,6 @@
     REGEX "Help/(dev|guide)($|/)" EXCLUDE
     )
 
-  # Install auxiliary files integrating with other tools.
-  add_subdirectory(Auxiliary)
-
   # Optionally sign installed binaries.
   if(CMake_INSTALL_SIGNTOOL)
     configure_file(Source/CMakeInstallSignTool.cmake.in Source/CMakeInstallSignTool.cmake @ONLY)
